Terminal.Gui/Windows/Dialog.cs

```diff
@@ -116,17 +116,104 @@ internal int GetButtonsWidth ()
 			}
 			return buttons.Select (b => b.Bounds.Width).Sum () + buttons.Count - 1;
 		}
+		/// <summary>
+		/// Determines the horizontal alignment of the Dialog buttons.
+		/// </summary>
+		public enum ButtonAlignments {
```

Comment:
The other alignment classes are not plural i.e. `TextAlignment` and `VerticalTextAlignment`.  So this would be tidier if it was just `ButtonAlignment` (sorry if this is pedantic).

I saw your issue around aligning controls more generally (which is cool).  Should we just call this `ViewAlignment` or just `Alignent` to future proof the public type? Or did you consider just using `TextAlignment` enum directly so there are no new enums?

Reply:
My opinion about this and supposing that this will be implemented for views, is to create a `ViewAlignment` enum on the `View` class, for users not make confusing with text, though they have the same values.

Reply:
The main reason for what I said is for the cases where you may want a `TextAlignment.Center` for text a `ViewAlignment.Left` for the views. If the same enum is used it's not very easy to distinguish them.

Reply:
> The main reason for what I said is for the cases where you may want a `TextAlignment.Center` for text a `ViewAlignment.Left` for the views. If the same enum is used it's not very easy to distinguish them.

But what I'm not proposing is for aligning a GROUP of views, not one. Pos/Dim work fine for doing that. So I don't think `ViewAlignment` is a good choice.

@tznind Unfortuantely `TextAlignment` is mis-named for this purpose. I called it `ButtonAlignmentS` because I wanted the property to be named `ButtonAlignment` and a class can't have both named the same thing.